changelogs/v249/skip_database_provision.md

```diff
@@ -0,0 +1,15 @@
+## Apps that use the `heroku/ruby` buildpack can now skip database provisioning on the first push with the  `HEROKU_SKIP_DATABASE_PROVISION` environment variable.
```

Comment:
It feels like this title might be a bit long for the changelog site (both from a field validation point of view, and from a legibility/consistency with other posts point of view).

What do you think about this shorter wording? (Feel free to use your own instead, if you can think of something better)

```suggestion
## Apps that use the `heroku/ruby` buildpack can now skip automatic database provisioning using an environment variable
```